layouts/footer.html

```diff
@@ -1,3 +1,3 @@
 </div>
-<div id="footer"><p>Copyright &copy; 2007-2013 The OpenRA Developers (<a href="/legal/">Legal notices</a>)</p></div>
-</div>
\ No newline at end of file
+<div id="footer"><p>Copyright &copy; 2007-2017 The OpenRA Developers (<a href="/legal/">Legal notices</a>)</p></div>
```

Comment:
I don't have nanoc installed locally to test this.
If you can verify it I'm all for making that change!